lib/base-command.js

```diff
@@ -75,7 +75,6 @@ class BaseCommand {
   }
 
   async setWorkspaces (filters) {
-    // TODO npm guards workspaces/global mode so we should use this.npm.prefix?
```

Comment:
is this TODO not something we still need to do? is this an answered question already?

Reply:
No, the entire approach to npm.prefix vs npm.localPrefix etc needs addressing and I'd rather leave these comments in until we actually properly audit and review how we're using those.